Core/ModuleInstaller.cs

```diff
@@ -252,18 +252,24 @@ public void InstallList(ICollection<CkanModule> modules, RelationshipResolverOpt
         ///
         /// Intended for previews.
         /// </summary>
-        // TODO: Return files relative to GameRoot
         public IEnumerable<string> GetModuleContentsList(CkanModule module)
         {
-            string filename = Cache.GetCachedZip(module);
+            string filename = Cache.GetCachedFilename(module);
 
             if (filename == null)
             {
                 return null;
             }
 
-            return FindInstallableFiles(module, filename, ksp)
-                .Select(x => x.destination);
+            try
+            {
+                return FindInstallableFiles(module, filename, ksp)
+                    .Select(x => ksp.ToRelativeGameDir(x.destination));
```

Comment:
I think `ToRelativeGameDir` will break `MainModInfo.OpenFileBrowser` unless a corresponding change is made there; double clicking the file contents attempts to open the node text in the file browser.

Reply:
Amended to the commit, this should address it.